### PR TITLE
Removing Ramda (from prod build) and going from 19k to 6k

### PR DESCRIPTION
**Pull request by jamonholmgren**
This PR removes Ramda from our production build of apisauce. (It's still present in devDependencies because I didn't feel like untangling the tests.)

```
            commonjs    umd
With Ramda:     7.6k  19.2k
Without:        6.6k   5.8k
```

As you can see, the size win is significant -- 1k less for commonjs and 13k less for umd (not sure why umd has such a huge win, but i'll take it).

At some point it would be nice to remove Ramda from the tests too, but that's a chore.

### CODE REVIEW

File: examples/github.js [hunk 1]
```diff
@@ -2,22 +2,18 @@ import apisauce from '../lib/apisauce'
...
 })
 
 // attach a monitor that fires with each request
 api.addMonitor(
-  R.pipe(
-    RS.dotPath('headers.x-ratelimit-remaining'),
-    R.concat('Calls remaining this hour: '),
-    console.log
-  )
+  R.pipe(RS.dotPath('headers.x-ratelimit-remaining'), R.concat('Calls remaining this hour: '), console.log),
```
Comment (bryanstearns) [inline examples/github.js:16]: I know this is automatic reformatting, but the Elixirist in me thinks that each pipe on its own line looks better!

Comment (jamonholmgren) [reply examples/github.js:16]: I agree! I might force it to do things differently here.

Comment (jamonholmgren) [reply examples/github.js:16]: I actually want to de-ramdaify this file as well, as I think it does more harm than good here.

**Review by bryanstearns — APPROVED** [review]
Looks good!

File: examples/github.js (hunk 1 above)
Comment (bryanstearns) [reply examples/github.js:16]: Probably not worth forcing the formatting - I was just nitpicking about how sometimes the automated tooling goofs

**Pull request merged**